lib/PluginFindRecord/PluginFindRecordModal.js

```diff
@@ -190,13 +196,21 @@ class PluginFindRecordModal extends React.Component {
     }
 
     const mixedResultsFormatter = {
-      isChecked: record => (
-        <Checkbox
-          type="checkbox"
-          checked={Boolean(checkedMap[record.id])}
-          onChange={() => this.toggleRecord(record)}
-        />
-      ),
+      isChecked: record => {
+        const value = getRecordLabel?.(record) ?? findIndex(records, ['id', record.id]);
```

Comment:
IMHO, it looks simpler with pseudo-code: `ariaLabel = getRecordLabel ? formatMessage(key, getRecordLabel()) : formatMesage(defaultKey, rowIndex);

Reply:
yep, missed that `MCL formatter` provides `rowIndex`.